tests/www/views/test_views_tasks.py

```diff
@@ -976,7 +957,7 @@ def test_task_instance_set_state_failure(admin_client, action):
     "task_search_tuples",
     [
         [("example_xcom", "bash_push"), ("example_bash_operator", "run_this_last")],
-        [("example_subdag_operator", "some-other-task")],
+        [("example_python_operator", "print_the_context")],
```

Comment:
I think we can get rid of this case entirely? It would just test the same thing as the bash one if the operator not a subdag one.